docs/compute-engine/api_page_templates/converter.md

```diff
@@ -7,7 +7,7 @@ The LCE converter allows you to convert a Keras model built with `larq` to an LC
 Before installing the LCE converter, please install:
 
 - [Python](https://www.python.org/) version `3.6`, `3.7`, or `3.8`
-- [Tensorflow](https://www.tensorflow.org/install) version `1.14`, `1.15`, `2.0.0`, `2.1.0`, or `2.2.0` (recommended):
+- [Tensorflow](https://www.tensorflow.org/install) version `1.14`, `1.15`, `2.0.0`, `2.1.0`, `2.2.0` or `2.3.0` (recommended):
```

Comment:
I think on the [frontpage](https://docs.larq.dev/#installation) we just list the minor versions, I think we should do the same here for consistency:
```suggestion
- [Tensorflow](https://www.tensorflow.org/install) version `1.14`, `1.15`, `2.0`, `2.1`, `2.2` or `2.3` (recommended):
```